Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -29,7 +62,14 @@ public void AddItem(Item item)
         /// </summary>
         public void RemoveItem(Item item)
```

Comment:
Since you already have the item object, you can call Items.Remove(item) rather than looping to find based on name. Let the list handle that for you :)

Reply:
That would result in Removing all objects that are sold even if i only want to sell 1?

Reply:
the `.Remove(T item)` method will only remove the first found instance. if there are duplicates it will not.

however, on a deeper level we should never really have duplicated objects. We might have "two swords" but they are not the _same_ sword.